nautobot/ipam/tests/test_api.py

```diff
@@ -601,3 +602,29 @@ def setUpTestData(cls):
                 "ports": [6],
             },
         ]
+
+    def test_ports_regression(self):
+        """
+        Test that ports can be provided as str or int.
+
+        Ref: https://github.com/nautobot/nautobot/issues/265
+        """
+        device = self.devices[0]
+
+        # Ports as string should be good.
+        data = {
+            "name": "http",
+            "protocol": "tcp",
+            "device": str(device.id),
+            "ports": ["80"],
+        }
+
+        self.add_permissions("ipam.add_service")
+        url = reverse("ipam-api:service-list")
+        response = self.client.post(url, data, format="json", **self.header)
+        self.assertHttpStatus(response, status.HTTP_201_CREATED)
```

Comment:
Can we check to make sure that the created record has correct `ports` values in both cases? Call me paranoid if you like. :-)

Reply:
Ah yes, thank.